js/models/serverConfigMd.js

```diff
@@ -0,0 +1,104 @@
+var Backbone = require('backbone'),
+    localStorageSync = require('../utils/backboneLocalStorage'),
+    is = require('is_js');
+
+module.exports = Backbone.Model.extend({
+  defaults: {
+    'id': 1,
+    'server_ip': 'localhost',
+    'rest_api_port': 18469,
+    'socket_port': 18466
+  },
+
+  sync: localStorageSync.sync,
```

Comment:
I'm using a very well known local storage adapter for backbone, so this model is saved to and fetched from local storage. The model manages the interface with local storage, so we shouldn't manually (outside of the model's interface) be accessing or modifying that data:

![image](https://cloud.githubusercontent.com/assets/794517/12736678/35a3d4ae-c907-11e5-9a7f-dccc3b4f7001.png)